mmd_tools/converters/bone_converter.py

```
@@ -0,0 +1,530 @@
+        skeleton_group = cmds.group(empty=True, name=SKELETON_GROUP, parent=root_group)
+
+        # ボーン名とインデックスのマッピングを作成
+        bone_map = self._create_bone_mapping(pmx_data.bones)
+
+        # Mayaジョイントを作成
+        maya_joints = self._create_maya_joints(
+            pmx_data.bones, bone_map, "pmx", skeleton_group
+        )
+
+        # リグのセットアップはRigConverterに委譲
+        rig_result = self.rig_converter.setup_pmx_rig(
```

Ruff (0.12.2)

Local variable `rig_result` is assigned to but never used

Remove assignment to unused variable `rig_result`

(F841)